resource-watch/resource-watch
# Improved styles and pagination in Dashboards "Add visualization" modal

## Overview
This PR includes the following improvements for the dashboard's widget edition modal:

1. The pagination toolbar is only displayed when there are two or more pages of widgets
2. The scroll has been updated so that it only applies to the widget list section

Scroll styles before:

![image](https://user-images.githubusercontent.com/545342/59605225-9f518400-910e-11e9-8373-5d362d050650.png)


Scroll styles after:

![image](https://user-images.githubusercontent.com/545342/59605160-76c98a00-910e-11e9-9d99-3fb926ca7083.png)


## Testing instructions
Go for instance to http://localhost:9000/myrw-detail/dashboards/302 and check the "Add visualization" modal.

## [Pivotal task](https://www.pivotaltracker.com/story/show/166730571)
